gcp/gcp.go

```diff
@@ -147,6 +147,7 @@ func (gcp *GoogleCloudPrint) Fetch(gcpID string) ([]Job, error) {
 	responseBody, errorCode, _, err := postWithRetry(gcp.robotClient, gcp.baseURL+"fetch", form)
 	if err != nil {
 		if errorCode == 413 {
+			log.Debugf("No jobs returned by fetch (413 error)")
```

Comment:
Sorry, I did not explicitly state that DebugPrinter takes another argument, the printer ID. I just checked for examples of log.[level]Printer, and all cases use the readable name of the printer, not the GCP ID, which is the only thing available here. Therefore, please forgive me and just call log.Debug or log.Debugf, and I'll merge.